implementations/ApiCatalog.json

```diff
@@ -13,7 +13,21 @@
     "options": {
       "type": "Ed25519Signature2020"
     },
-    "tags": ["vc-api", "Ed25519Signature2020", "vc2.0", "eddsa-2022", "ecdsa-2019" ]
+    "tags": ["vc-api", "Ed25519Signature2020", "vc2.0"]
+  }, {
+    "id": "did:key:z6Mkska8oQD7QQQWxqa7L5ai4mH98HfAdSwomPFYKuqNyE2y",
+    "endpoint": "https://vc.apicatalog.com/credentials/issue",
+    "options": {
+      "type": "eddsa-2022"
+    },
+    "tags": ["vc-api", "eddsa-2022"]
+  }, {
+    "id": "did:key:z6Mkska8oQD7QQQWxqa7L5ai4mH98HfAdSwomPFYKuqNyE2y",
+    "endpoint": "https://vc.apicatalog.com/credentials/issue",
+    "options": {
+      "type": "ecdsa-2019"
```

Comment:
Same comment as above... we shouldn't change the parameters that implementers take w/o their consent... @filip26 -- how would you like to receive these options to put a different type of signature on the VCs you create?

Reply:
I now see that these parameters were requested by @filip26 in https://github.com/w3c-ccg/vc-api-test-suite-implementations/pull/69